### PR TITLE
(maint) Bump vanagon to 0.8.1

### PR DESCRIPTION
**Pull request by ScottGarman**
This is needed to fix PA-569, by enabling the brp strip scripts on Cisco RPM platforms. There is an unrelated bug in vanagon that breaks windows builds, and a workaround for this to get CI green again so we can then take a closer look at the vanagon issue, which doesn't appear to be trivial.

### CODE REVIEW

**Comment by ScottGarman** [comment]
Currently testing this with a jenkins p-a ad-hoc build, will post a link to a green build when ready.

**Comment by ScottGarman** [comment]
Fully green ad-hoc build at: https://jenkins.puppetlabs.com/view/puppet-agent/view/ad%20hoc/job/platform_puppet-agent_pkg-van-ship_manual-ad-hoc/362/

**Comment by ScottGarman** [comment]
Note: I've filed VANAGON-14 to ticket the vanagon issue that needs further examination/work.

**Comment by branan** [comment]
This is kinda hacky, but that's a small enough file that I guess we're gonna be OK throwing it into git for all eternity. Agreed that VANAGON-14 needs to be resolved, though

File: configs/components/ruby.rb [hunk 1]
```diff
@@ -4,9 +4,9 @@
   pkg.url "https://cache.ruby-lang.org/pub/ruby/2.3/ruby-#{pkg.get_version}.tar.gz"
 
   if platform.is_windows?
-    pkg.add_source "http://buildsources.delivery.puppetlabs.net/windows/elevate/elevate.exe", sum: "bd81807a5c13da32dd2a7157f66fa55d"
+    pkg.add_source "file://resources/files/windows/elevate.exe", sum: "bd81807a5c13da32dd2a7157f66fa55d"
```
Comment (stahnma) [inline configs/components/ruby.rb]: I don't remember if we looked at licensing for this. Can we redistribute it? At this point, it's part of vanagon, which feels weird to me.

Comment (McdonaldSeanp) [inline configs/components/ruby.rb]: Does it matter whether or not it's in the vanagon repo? we have been distributing it as part of the package for a while now, going back to puppet_for_the_win, and it's been part of the license file: https://github.com/puppetlabs/puppet_for_the_win/blob/be067d5c6456dba22f30938abaeeded29004bb3b/conf/windows/stage/misc/LICENSE.rtf#L12

Comment (ScottGarman) [inline configs/components/ruby.rb]: @stahnma don't we redistribute the binary in p-a packages?

Comment (ScottGarman) [inline configs/components/ruby.rb]: Also, it's not part of vanagon, just the puppet-agent repo.

**Comment by ScottGarman** [comment]
I managed to fix the issue in vanagon, so I'll close this PR once I cut a new vanagon release and put up a replacement PR...

**Comment by ScottGarman** [comment]
Closing this now in favor of PR 867.

**Pull request closed without merging**